docs/components/input.md

```diff
@@ -215,43 +216,58 @@ You can pass any input attribute and it will get properly bound to the element.
 
 KInput works as regular inputs do using v-model for data binding:
 
-<Komponent :data="{myInput: 'hello'}" v-slot="{ data }">
-  <div>
-    {{ data.myInput }}
-    <KInput
-      v-model="data.myInput"
-      @blur="e => (data.myInput = 'blurred')" />
+<div>
+  {{ myInput }}
```

Comment:
Thinking we can take out line 220 and  222 here, as I think they were probably mostly to demo this bug? To me having the clear button is nice however.

Reply:
Yep, I want to keep this in just to make sure we have the scenario in the docs so we don't break it again!